resources/contents/blog/en/review-2018.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 201809211
+title: "Looking back at 2018"
+date: 2018-12-23T15:27:25+02:00
+author: Rathes Sachchithananthan
+template: post
+description: Looking back at an incredible year 2018 and planning things for another upcoming great year
+image: /images/blog/review-2018.jpg
+categories:
+  - Life
+tags:
+  - review
+locale: en_US
+---
+
+By the end of every year, I spend some time looking back at the year that just passed and evaluate how well I did. Did I reach goals? Did I feel good throughout the year and how do I think about the year? I also take the review as an ooportunity to adjust long term goals and goals for the upcoming year.
+This year had some major changes and therefore this year feels like it ended too fast. When I look back at my last years review and what I set as a main focus for this year it was more about doing less, more about investing in people and relationships than on work. I decided to drop side-projects and I set the goals to get into public speaking and open source development.
+I started that way. In the beginning of 2018 I got in touch with some people that I hadn't spoken to for a long time (like for years). I also started publishing some work to open source.
+
+## From Ghent with love
+
+But during March/April there was a big chance coming towards me. In a sub-clause of my review I mentioned that I wanted to move to a Dutch speaking area and work there for a while. And by the end of March I got that opportunity: I got the opportunity to work at Teamleader in the beautiful city of Ghent. And without wasting too much time I took this chance and moved to Gent in May. [I even wrote about it here](https://rathes.me/blog/en/from-ghent-with-love)
+That change destroyed my other plans. Instead of getting back in touch with people that I hadn't talked too much I met and still meet new people. For the first time in my life I'm living longer than 6 months in another country than Germany and I am pretty sure that I'm not going to move back. My plans to continue learning Japanese have been delayed, instead I'm trying to survive in an environment with a lot of Flemish accents and the amount of work (as an employee) definitely increased compared to last year.
```

Comment:
```suggestion
That change destroyed my other plans. Instead of getting back in touch with people that I hadn't talked too much I met and still meet new people. For the first time in my life, I'm living longer than 6 months in another country than Germany and I am pretty sure that I'm not going to move back. My plans to continue learning Japanese have been delayed, instead, I'm trying to survive in an environment with a lot of Flemish accents and the amount of work (as an employee) definitely increased compared to last year.
```